src/main/java/services/taskpresentation/TaskGetter.java

```diff
@@ -31,4 +32,20 @@ public TaskInfo getTaskById(Long id) {
         return null;
     }
 
+    /**
+     * Get all tasks.
+     * @return a list of TaskInfo
+     */
+    @Override
+    public List<TaskInfo> getTasks() {
```

Comment:
Seems there is no corresponding update in Task controller

Reply:
Yup because `TaskController` did not use this method

Reply:
OK so this pr is just about refactoring